packages/app-defaults/src/createApp.test.tsx

```diff
@@ -33,7 +35,23 @@ describe('Optional ThemeProvider', () => {
       ),
     };
 
-    const App = createApp({ components }).getProvider();
+    const __experimentalI18n = {
```

Comment:
Let's drop this example and the user-settings dep

Reply:
ok, I think, I misunderstand your comment above